cam2imageGUI.py

```diff
@@ -39,7 +39,9 @@
 }
 
 
+#####################
 ### Error Windows ###
+#####################
 
 def missingPath():

```

Comment:
def missing_path() -> None:

Your naming convention is not the one used in python, functions/variables are linked with _